src/main/java/edu/hm/hafner/coverage/FileNode.java

```diff
@@ -197,11 +197,13 @@ private void mergeCounters(final FileNode otherFile) {
                     left.clearMissed();
                     left.setTotalFromCovered();
                 }
-                else {
-                    throw new IllegalArgumentException(
-                            String.format(Locale.ENGLISH, "Cannot merge coverage information for line %d in %s",
-                                    line, this));
-                }
+                // Commenting out the following lines to avoid throwing an exception when merging coverage information
+                // from Jacoco. This is a temporary solution until we have a better way to handle this.
+//                else {
+//                    throw new IllegalArgumentException(
```

Comment:
Throwing the exception was interruption the coverage parsing for jacoco parser. So, we commented it out and added necessary unit tests to make sure the coverage was calculated as expected.